minerl/herobraine/env_spec.py

```diff
@@ -53,7 +53,9 @@ def reset(self):
 
         # after create_server_world_generators(), because it will see python generated map
         # to pick a good location
-        self.agent_start = [self.create_agent_start() for _ in range(self.agent_count)]

```

Comment:
I think this is equivalent to self.agent_start = [self.create_agent_start() for self.current_agent in range(self.agent_count)]

Reply:
Yeah good point that would probably work too. Somehow assigning a `self.<something>` made me feel like it wouldn't work so I switched it lol. Maybe not worth the effort to change though.